packages/common/src/utils/TypeUtil.ts

```diff
@@ -0,0 +1,68 @@
+export type TransactionStatus = 'confirmed' | 'failed' | 'pending'
+
+export type TransactionDirection = 'in' | 'out' | 'self'
+
+export type TransactionImage = {
+  type: 'FUNGIBLE' | 'NFT' | undefined
```

Comment:
```suggestion
  type?: 'FUNGIBLE' | 'NFT'
```

Reply:
I think making a prop optional and making it undefined'able is not the same. `?` will reach the same functionality in tye TS worlds. But the meaning will be different